modules/cloudtrail/log-metrics.tf

```diff
@@ -0,0 +1,99 @@
+locals {
+  metrics_patterns_mapping = {
+    "iam-user" = {
```

Comment:
lets change naming to have action also included not just service, something like "iam-user-delete", "iam-role-delete", "iam-policy-change", s3-modify and etc